recipes/stc/all/conanfile.py

```diff
@@ -0,0 +1,25 @@
+import os
+from conans import ConanFile, tools
+
+
+class StcConan(ConanFile):
+    name = "stc"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/tylov/STC"
+    description = "A modern, user friendly, generic, type-safe and fast C99 container library: String, Vector, Sorted and Unordered Map and Set, Deque, Forward List, Smart Pointers, Bitset and Random numbers."
+    topics = ("conan", "stc", "containers", "string", "vector", "map", "set", "deque", "bitset", "random", "list")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy("*.h", dst="include", src=self._source_subfolder, keep_path=True)
+        self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
+
+    def package_id(self):
+        self.info.header_only()
```

Comment:
There is no `no_copy_source`.

Wonder how it passes the hook 'HEADER_ONLY, NO COPY SOURCE (KB-H005)'

Reply:
@uilianries do you remember why `no_copy_source` was required?
it seems like recipe(s) work just fine without it. doc says:
```
This is mostly an optimization for packages with large source codebases, to avoid extra copies.
```
so it should be mostly useful for packages like boost or Qt.

Reply:
> @uilianries do you remember why no_copy_source was required?

We don't need to copy to build folder here. This recipe only downloads to source folder and copy to package folder, that's it. No need extra step with build folder.

Reply:
> so it should be mostly useful for packages like boost or Qt.

Boost and Qt are huge! We don't want to explode our storage with those packages. It's a different case.

Reply:
> Wonder how it passes the hook 'HEADER_ONLY, NO COPY SOURCE (KB-H005)'

Because is a warning, not an error:

> [HOOK - conan-center.py] pre_export(): WARN: [HEADER_ONLY, NO COPY SOURCE (KB-H005)] This recipe is a header only library as it does not declare 'settings'. Please include 'no_copy_source' to avoid unnecessary copy steps

Reply:
okay, so it's not a strict requirement per hook, but more like a recommendation for an optimization.
I've added `no_copy_source`, probably the benefit is very small and even unnoticeable for such a tiny project.

Reply:
> > Wonder how it passes the hook 'HEADER_ONLY, NO COPY SOURCE (KB-H005)'
> 
> Because is a warning, not an error:

Ah, got it, I always have a `CONAN_HOOK_ERROR_LEVEL=30` while the CCI seems to have it set to `40`.